### DISCUSSION TITLE
Request for running Llama 2 with Nemo

### DISCUSSION THREAD

**Discussion by LouieBHLu** [NVIDIA-NeMo/NeMo#7718] [General]
Hi,

Is there any tutorial or example to run LLama2 with Nemo?

Thank you.

**Comment by titu1994** [comment]
The following tutorials may help with using LLama 2 models in NeMo - 

Llama2 SFT - https://docs.nvidia.com/nemo-framework/user-guide/latest/playbooks/llama2sft.html 
Llama2 PEFT - https://docs.nvidia.com/nemo-framework/user-guide/latest/playbooks/llama2peft.html

**Reply by LouieBHLu** [reply]
Wow, thank you for your reply. This is very helpful.

**Reply by alex-ht** [reply]
Hi @titu1994
Why is it that the entire process does not require specifying a chat template?
https://github.com/NVIDIA/NeMo/blob/32e630220d2c24550a869f43ce618949ccf1a1a5/examples/nlp/language_modeling/tuning/conf/megatron_gpt_sft.yaml#L110
Do we need to specify the prompt_template: '[INST] {input} [/INST] {output}'?

**Reply by titu1994** [reply]
@cuichenx @arendu
